bluepyemodel/evaluation/evaluator.py

```diff
@@ -452,7 +457,8 @@ def define_optimisation_protocol(
         stochasticity (bool): Should the stochastic channels be stochastic or
             deterministic
         efel_settings (dict): eFEl settings.
-        use_fixed_dt_recordings (bool): whether to record at a fixed dt of 0.1 ms.
+        use_fixed_dt_recordings (bool): used for legacy currentscape 
+            to record at a fixed dt of 0.1 ms.
     """
 
     protocols = define_protocols(
```

Comment:
Could you also pass dt to this function?

Reply:
done!